media/css/firefox/new/redesign/common.less

```diff
@@ -188,3 +181,20 @@ main {
         opacity: 0.9;
     }
 }
+
+/* hide all download related links in the footer initially */
+.fx-footer-links.os_android,
+.fx-footer-links.os_ios,
+.fx-footer-links.os_desktop {
+    display: none;
+}
+
+/* show links based on thge current os */
+.linux .fx-footer-links.os_desktop,
+.linux.x86.x64 .fx-footer-links.os_desktop,
+.osx .fx-footer-links.os_desktop,
+.windows .fx-footer-links.os_desktop,
+.android .fx-footer-links.os_android,
+.ios .fx-footer-links.os_ios {
+    display: block;
+}
```

Comment:
Should this display logic be made into an include or mixin? You're much more familiar with this entire button system than I, so definitely your call. Just thought I'd ask. :smile: